src/diffusers/pipelines/chroma/pipeline_chroma.py

```diff
@@ -233,20 +233,21 @@ def _get_t5_prompt_embeds(
             return_tensors="pt",
         )
         text_input_ids = text_inputs.input_ids
-        attention_mask = text_inputs.attention_mask.clone()
+        tokenizer_mask = text_inputs.attention_mask
```

Comment:
Could we add some comment on how `tokenizer_mask` and `attention_mask` are differentiated from one another and which one is passed where?